packages/blade/src/components/Table/componentIds.ts

```diff
@@ -12,6 +12,7 @@ const ComponentIds = {
   TableFooter: 'TableFooter',
   TableFooterRow: 'TableFooterRow',
   TableFooterCell: 'TableFooterCell',
+  VirtualizedTable: 'VirtualizedTable',
```

Comment:
This doesn't exist right?

Reply:
it is being used in TableVirtualizedWrapper